resources/home/dnanexus/utils/io.py

```
@@ -22,8 +28,11 @@
     targets_url,
     job_output,
     expiry_date,
+    build,
```

Ruff (0.8.0)

Missing type annotation for function argument `build`

(ANN001)